### ISSUE TITLE
Classic World of Warcraft

### ISSUE THREAD

**Issue by jmckay85** [Wutname1/SpartanUI#415]
I installed SpartanUI and loaded the game, went through the UI setup wizard. Then downloaded more addons for WoW and then restarted WoW. After I restarted I could not get WoW to connect. It kept giving me a connection error. On top of that I was in a voice chat on battle.net and it would disconnect my battle.net as well and tell me to go online to chat and play, everything else had internet.  I tried restarting my pc, resetting my modem, resetting my network settings on my pc, repairing the game, uninstalling and reinstalling battle.net, followed a few forums on how to fix the connection problem and none of them worked. I finally decided to uninstall the addons I had downloaded one at a time to see if any of them were causing the issue, after uninstalling all of them, I only had SpartanUI left to uninstall. Once I did that WoW loaded just fine. Reinstalled SpartanUI thinking that I had a corrupt file and a reinstall would solve it....it did not. My best friend was playing WoW with me on his computer, downloaded the same addons and didn't have any issues what so ever. I have a i7 9th gen CPU, 8g ram, Nvidia GEForce GTX 1650 graphics...His system is similar to mine. Anyone have any ideas on how to fix this problem? Could it possibly be a setting in the setup wizard? Please help! I really love SpartanUI and would like to use it. Thanks in advance.

**Comment by Wutname1** [comment]
That is definitely a weird experience and I've never seen it before. Can you not launch the game at all or can you not sign in to a character? Add on settings are not loaded until you actually sign in to a character. If the add-on existing is keeping the game itself from opening then it has to be something related to the files on your computer, and only blizzard support would really be able to tell you more.

**Comment by jmckay85** [comment]
I cannot launch the game at all. I reinstalled SpartanUI last night and it worked just fine. I played for a couple of hours. Then this morning I went to load the game and got the same connection message. So I went and uninstalled SpartanUI again, Still didn't work. I uninstalled all the addons and it still isn't working. Not entirely sure what to do now. Going to start from scratch and try all of the things I did before. I may even uninstall WoW and then reinstall as well. 
![Connection issue to wow](https://user-images.githubusercontent.com/81725241/113302259-7e4b2a80-92c5-11eb-9420-5cfd699af6ef.png)
